invenio_vocabularies/alembic/e1146238edd3_create_awards_table.py

```diff
@@ -0,0 +1,57 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Create awards table."""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'e1146238edd3'
+down_revision = '3d362de4926a'
```

Comment:
Should the revision be `8ff82dfb0be8` (vocabularies branch)? In theory `awards` does not need funders (i.e. in case all funders are **not** from a vocabulary). In that case, it should just be dependent on the start of vocabularies.